src/systems/physics/Physics.cc

```diff
@@ -2869,8 +2869,8 @@ void PhysicsPrivate::UpdateSim(EntityComponentManager &_ecm,
   IGN_PROFILE_END();
 
   // pose/velocity/acceleration of non-link entities such as sensors /
-  // collisions. These get updated only if another system has created a
-  // components::WorldPose component for the entity.
+  // collisions. These get updated only if another system has created
+  // the component for the entity.
```

Comment:
Is this change accurate, or out of place? I don't see any changes to the physics code, just a change to the comment

Reply:
I found out the comment was wrong while working on this PR and it actually took me some back-and-forth to realize it. The way it's originally written, I thought that I had to create a `WorldPose` component in order to have the `WorldLinearVelocity` filled, which turned out not to be true. When in fact, the components listed below are decoupled from each other.

So the change should be accurate, and it's related, even though I recognize it isn't obvious at a first glance.